spec/dockerfile_spec.rb

```diff
@@ -21,11 +21,11 @@
   end
 
   describe command('node -v') do
-    its(:stdout) { should match /4\.2\.2/ }
+    its(:stdout) { should match /6\.9\.2/ }
   end
 
   describe command('npm -v') do
-    its(:stdout) { should match /2\.\d{1,2}\.\d{1,2}/ }
+    its(:stdout) { should match /3\.\10\.\9/ }
```

Comment:
you don't need to change the minor and patch version number matching if you just want to verify if the npm is `v3.x.x`